aggregator/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJob.java

```diff
@@ -34,24 +37,27 @@
     @Inject
     EndpointEmailSubscriptionResources subscriptionResources;
 
-    private static final String DEFAULT_STRING_SERIALIZER = "org.apache.kafka.common.serialization.StringSerializer";
-    private static final String DEFAULT_BROKER_URL = "localhost:9092";
-    private static final String DEFAULT_ACKS = "1";
+    @ConfigProperty(name = "quarkus.aggregator.serializer")
+    String serializer;
 
-    @Scheduled(identity = "dailyEmailProcessor", cron = "{email.subscription.daily.cron}")
-    public void processDailyEmail(ScheduledExecution se) {
-        // Only delete on the largest aggregate time frame. Currently daily.
-        final List<EmailAggregation> aggregatedEmails = processAggregateEmails(se.getScheduledFireTime());
+    @ConfigProperty(name = "quarkus.aggregator.broker-url")
+    String brokerUrl;
 
-        Map<String, String> config = new HashMap<>();
-        config.put("bootstrap.servers", DEFAULT_BROKER_URL);
-        config.put("key.serializer", DEFAULT_STRING_SERIALIZER);
-        config.put("value.serializer", DEFAULT_STRING_SERIALIZER);
-        config.put("acks", DEFAULT_ACKS);
+    @ConfigProperty(name = "quarkus.aggregator.acks")
+    String acks;
+
+    @ConfigProperty(name = "quarkus.aggregator.topic.name")
+    String topicName;
+
+//    @Scheduled(identity = "dailyEmailProcessor", cron = "{email.subscription.daily.cron}")
+    @Scheduled(identity = "dailyEmailProcessor", every = "1m")
```

Comment:
Looks like this was used for testing ? I guess the line above would be the correct one ?

Reply:
Yes, I don't know how else I can test if the cron ob feature is working otherwise.
I would remove all of these things when the backend part is ready and it's working together as expected.

Reply:
You can always override the EMAIL_SUBSCRIPTION_DAILY_CRON environment variable on your machine

Reply:
Fixed.